### PR TITLE
Lasb 623 update v1 get hearings swagger doc

### PR DESCRIPTION
**Pull request by chris-groves**
## What

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=454&projectKey=LASB&modal=detail&selectedIssue=LASB-623

Update the API documentation for V1 GET hearings endpoint so users know what they will receive when querying this endpoint

## Checklist

Before you ask people to review this PR:

- [ ] Tests and linters should be passing
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: swagger/v1/hearing.json [hunk 1]
```diff
@@ -278,12 +286,12 @@
         },
         "listingSequence": {
           "description": "Sequence number for the hearing",
-          "example": 1,
+          "example": "1",
```
Comment (arthurashman) [inline swagger/v1/hearing.json:289]: If the type is number, should these be left as Ints?

File: swagger/v1/hearing.json [hunk 2]
```diff
@@ -278,12 +286,12 @@
...
           "type": "number"
         },
         "listedDurationMinutes": {
           "description": "Duration of the hearing in minutes",
-          "example": 120,
+          "example": "120",
```
Comment (arthurashman) [inline swagger/v1/hearing.json:294]: If the type is number, should these be left as Ints?

File: swagger/v1/hearing.json (hunk 1 above)
Comment (chris-groves) [reply swagger/v1/hearing.json]: Good spot - have corrected

File: swagger/v1/hearing.json (hunk 2 above)
Comment (chris-groves) [reply swagger/v1/hearing.json]: Good spot - have corrected

**Review by arthurashman — APPROVED** [review]

**Pull request merged**